### PR TITLE
id_ID v7 translation

### PR DESCRIPTION
**Pull request by WilloIzCitron**
If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [ ] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [ ] I have ensured that my code compiles, if applicable.
- [ ] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

File: core/assets/bundles/bundle_in_ID.properties [hunk 1]
```diff
@@ -1870,94 +1870,94 @@ lenum.build = Membangun sebuah sttruktur.
...
-block.blast-door.name = Blast Door
-block.carbide-wall.name = Carbide Wall
-block.carbide-wall-large.name = Large Carbide Wall
+block.yellowcoral.name = Koral Kuning
+block.carbon-boulder.name = Batu Besar Karbon
+block.ferric-boulder.name = Batu Besar Besi
+block.beryllic-boulder.name = Batu Besar Berilik
+block.yellow-stone-boulder.name = Batu Besar Kuning
+block.arkyic-boulder.name = Batu Besar Arkyik
+block.crystal-cluster.name = Gugus Kristal
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:1917]: Bongkahan Kristal

File: core/assets/bundles/bundle_in_ID.properties [hunk 2]
```diff
@@ -1870,94 +1870,94 @@ lenum.build = Membangun sebuah sttruktur.
...
+block.vibrant-crystal-cluster.name = Gugus Kristal yang Bersinar
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:1918]: Bongkahan Kristal yang Bersinar

File: core/assets/bundles/bundle_in_ID.properties [hunk 3]
```diff
@@ -1870,94 +1870,94 @@ lenum.build = Membangun sebuah sttruktur.
...
-block.phase-synthesizer.name = Phase Synthesizer
-block.heat-reactor.name = Heat Reactor
-block.beryllium-wall.name = Beryllium Wall
-block.beryllium-wall-large.name = Large Beryllium Wall
-block.tungsten-wall.name = Tungsten Wall
-block.tungsten-wall-large.name = Large Tungsten Wall
-block.blast-door.name = Blast Door
-block.carbide-wall.name = Carbide Wall
-block.carbide-wall-large.name = Large Carbide Wall
+block.yellowcoral.name = Koral Kuning
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:1911]: Karang Kuning; Terumbu Kuning

File: core/assets/bundles/bundle_in_ID.properties [hunk 4]
```diff
@@ -1156,23 +1156,23 @@ item.pyratite.name = Pyratit
...
+liquid.neoplasm.name = Neoplasma
 liquid.arkycite.name = Arkycite
 liquid.gallium.name = Gallium
-liquid.ozone.name = Ozone
-liquid.hydrogen.name = Hydrogen
+liquid.ozone.name = ozon
+liquid.hydrogen.name = Hidrogen
 liquid.nitrogen.name = Nitrogen
-liquid.cyanogen.name = Cyanogen
+liquid.cyanogen.name = Cianogen
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:1175]: Sianogen

File: core/assets/bundles/bundle_in_ID.properties [hunk 5]
```diff
@@ -1136,7 +1136,7 @@ content.unit.name = Unit
 content.block.name = Blok
 content.status.name = Status Efek
 content.sector.name = Sektor
-content.team.name = Factions
+content.team.name = Faksi
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:1139]: Fraksi

File: core/assets/bundles/bundle_in_ID.properties [hunk 6]
```diff
@@ -1870,94 +1870,94 @@ lenum.build = Membangun sebuah sttruktur.
...
+block.ferric-stone.name = Batu Besi
+block.ferric-craters.name = Kawah Besi
+block.beryllic-stone.name = Batu Berilik
+block.crystalline-stone.name = Batu Kristal
+block.crystal-floor.name = Lantai Kristal
+block.yellow-stone-plates.name = Plat Batu Kuning
+block.red-stone.name = Batu Merah
+block.dense-red-stone.name = Batu Merah Padat
+block.red-ice.name = Es Merah
+block.arkycite-floor.name = Lantai Arkycit
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:1888]: Lantai Arkisit

File: core/assets/bundles/bundle_in_ID.properties [hunk 7]
```diff
@@ -1870,94 +1870,94 @@ lenum.build = Membangun sebuah sttruktur.
...
+block.arkyic-stone.name = Batu Arkyic
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:1889]: Batu Arkyik

File: core/assets/bundles/bundle_in_ID.properties [hunk 8]
```diff
@@ -1870,94 +1870,94 @@ lenum.build = Membangun sebuah sttruktur.
...
+block.rhyolite-vent.name = Ventilasi Riolit
+block.carbon-vent.name = Ventilasi Karbon
+block.arkyic-vent.name = Ventilasi Arkyic
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:1892]: Ventilasi Arkyik

File: core/assets/bundles/bundle_in_ID.properties [hunk 9]
```diff
@@ -1870,94 +1870,94 @@ lenum.build = Membangun sebuah sttruktur.
...
-block.red-stone-wall.name = Red Stone Wall
-block.red-diamond-wall.name = Red Diamond Wall
+block.core-zone.name = Zona inti
+block.regolith-wall.name = Dinding Regolit
+block.yellow-stone-wall.name = Dinding Batu kuning
+block.rhyolite-wall.name = Dinding Riolit
+block.carbon-wall.name = Dinding Karbon
+block.ferric-stone-wall.name = Dinding Batu Besi
+block.beryllic-stone-wall.name = Dinding Batu Berilik
+block.arkyic-wall.name = Dinding Arkyic
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:1904]: Dinding Arkyik

File: core/assets/bundles/bundle_in_ID.properties [hunk 10]
```diff
@@ -572,17 +572,17 @@ research.multiplayer = Hanya host yang dapat meneliti barang.
...
-objective.destroyblocks = [accent]Destroy: [lightgray]{0}[white]/{1}\n{2}[lightgray]{3}
-objective.item = [accent]Obtain: [][lightgray]{0}[]/{1}\n{2}[lightgray]{3}
-objective.coreitem = [accent]Move into Core:\n[][lightgray]{0}[]/{1}\n{2}[lightgray]{3}
-objective.build = [accent]Build: [][lightgray]{0}[]x\n{1}[lightgray]{2}
+objective.research = [accent]Riset:\n[]{0}[lightgray]{1}
+objective.produce = [accent]Memperoleh:\n[]{0}[lightgray]{1}
+objective.destroyblock = [accent]Hancurkan:\n[]{0}[lightgray]{1}
+objective.destroyblocks = [accent]Hancurkan: [lightgray]{0}[white]/{1}\n{2}[lightgray]{3}
+objective.item = [accent]Memperoleh: [][lightgray]{0}[]/{1}\n{2}[lightgray]{3}
+objective.coreitem = [accent]Pindak ke Inti:\n[][lightgray]{0}[]/{1}\n{2}[lightgray]{3}
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:580]: Pindahkan ke Inti

**Review by DaGamerFiles — CHANGES_REQUESTED** [review]
🗿

**Review by DaGamerFiles — CHANGES_REQUESTED** [review]
Capitalization on line 406, 866, 1089, 1102

**Comment by WilloIzCitron** [comment]
done ez commit

File: core/assets/bundles/bundle_in_ID.properties [hunk 11]
```diff
@@ -1870,143 +1870,143 @@ lenum.build = Membangun sebuah sttruktur.
...
+block.reinforced-liquid-tank.name = Tangki Cairan yang Diperkuatkan
+block.beam-node.name = Simpul Sinar
+block.beam-tower.name = Menara Sinar
+block.beam-link.name = Tautan Sinar
+block.turbine-condenser.name = Kondensor Turbin
+block.chemical-combustion-chamber.name = Ruang Pembakaran Kimia
+block.pyrolysis-generator.name = Generator Pirolisis
+block.vent-condenser.name = Kondensor Ventilasi
+block.cliff-crusher.name = Penghancur Tebing
 block.plasma-bore.name = Plasma Bore
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:1974]: Bor Plasma

File: core/assets/bundles/bundle_in_ID.properties [hunk 12]
```diff
@@ -1870,143 +1870,143 @@ lenum.build = Membangun sebuah sttruktur.
...
-block.large-plasma-bore.name = Large Plasma Bore
-block.impact-drill.name = Impact Drill
-block.eruption-drill.name = Eruption Drill
-block.core-bastion.name = Core Bastion
-block.core-citadel.name = Core Citadel
-block.core-acropolis.name = Core Acropolis
-block.reinforced-container.name = Reinforced Container
-block.reinforced-vault.name = Reinforced Vault
+block.large-plasma-bore.name = Plasma Bore Besar
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:1975]: Bor Plasma Besar

File: core/assets/bundles/bundle_in_ID.properties [hunk 13]
```diff
@@ -1870,143 +1870,143 @@ lenum.build = Membangun sebuah sttruktur.
...
+block.world-cell.name = Sel Dunia
+block.shield-breaker.name = Penghancur Pelindung (nama/sprite sementara)
+block.tank-fabricator.name = Fabrikator Tank
+block.mech-fabricator.name = Fabrikator Mech
+block.ship-fabricator.name = Fabrikator Kapal
+block.refabricator.name = Refabrikator
+block.prime-refabricator.name = Refabrikator Perdana
+block.unit-repair-tower.name = Menara Perbaikan Unit
+block.diffuse.name = Membaur
+block.basic-assembler-module.name = Modul Assembler Dasar
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_in_ID.properties:2009]: Modul Perakit Dasar

**Review by DaGamerFiles — CHANGES_REQUESTED** [review]

**Comment by DaGamerFiles** [comment]
too bad another changes to do lol

**Comment by WilloIzCitron** [comment]
~~shift button on my keyboard succ~~

**Review by DaGamerFiles — APPROVED** [review]

**Comment by Anuken** [comment]
Merged.

**Pull request closed without merging**